gpflow/core/compilable.py

```diff
@@ -89,6 +89,8 @@ def __init__(self, *args, **kwargs):
             if autobuild_on and global_autobuild_on:
                 self.build()
                 self.initialize(force=True)
+        if origin_init.__doc__:
+            __init__.__doc__ = '\n'.join([origin_init.__doc__, __init__.__doc__])
```

Comment:
Guess, replacing will be just fine. You don't really need docstring of autobuild in descendant.

Reply:
Well, the autobuild parses the kwargs before passing them on, so might be useful to keep that in the docstring, no?

Reply:
autobuild doesn't have any docstrings

Reply:
It does: core/compilable.py:73: ```            """
            The `kwargs` may or may not contain 'autobuild' option. This option is
            inherited implicitly by all classes.
            """
```

Reply:
That's the thing that was overwriting the Model `__init__` docstrings in the first place.